src/Blackjack/Blackjack.Client.Console/Program.cs

```diff
@@ -17,108 +17,15 @@ static void Main(string[] args)
-                new Card("2", Suite.Hearts, 2, true),
-                new Card("7", Suite.Hearts, 7, true)};
-
-
-            Deck deck = new Deck(cards);
-            List<Card> shuffledList = deck.Shuffle();
-
-             totalMe = +shuffledList[0].Points + shuffledList[1].Points;
-             totalHouse = +shuffledList[2].Points + shuffledList[3].Points;
-         
-            Deck shuffledDeck = new Deck(shuffledList);
-
-
-
-            Console.WriteLine($"You have been dealt: {GetCardDercription(shuffledDeck.Next())}, {GetCardDercription(shuffledDeck.Next())}");
-            Console.WriteLine($"House has been dealt: {GetCardDercription(shuffledDeck.Next())}, {GetCardDercription(shuffledDeck.Next())}");
-            Console.WriteLine();
-            Console.WriteLine("What do you want to do? ");
-            Console.WriteLine("Choose [1] to take another card");
-            Console.WriteLine("Choose [2] to finish");
-            Console.WriteLine();
-            bool correct = false;
-
-            while (!correct)
-            {
-                string option = Console.ReadLine();
-                Console.WriteLine("I choose: " + option);
-                Console.WriteLine();
-
-                if (option == "1")
-                {
-                    var betsUp = shuffledDeck.Next();
-                    Console.WriteLine($"You have been dealt: {GetCardDercription(betsUp)}");
-                    var addPoints = betsUp.Points;
-                    totalMe = totalMe + addPoints;
-
-                    var housePoints = shuffledDeck.Next();
-                    Console.WriteLine($"House has been dealt: {GetCardDercription(housePoints)}");
-                    var addHousePoints = housePoints.Points;
-                    totalHouse = totalHouse + addHousePoints;
-
-                    Console.WriteLine();
-                    Console.WriteLine("What do you want to do? ");
-                    Console.WriteLine("Choose [1] to take another card");
-                    Console.WriteLine("Choose [2] to finish");
-                    Console.WriteLine();
-                    continue;
-                }
-                else
-                {
-                    correct = true;
-                    Console.WriteLine("You have " + totalMe + " points vs house have " + totalHouse + " points");
-                    if(totalMe == 21)
-                    {
-                        Console.WriteLine("You win!");
-                        Console.WriteLine();
-                    }
-
-                    else if (totalMe > totalHouse && totalMe <= 21)
-                    {
-                        Console.WriteLine("You win!");
-                        Console.WriteLine();
-                    }
-                    else
-                    {
-                        Console.WriteLine("House win!");
-                        Console.WriteLine();
-
-                    }
-                }
-            }
             
-
+            Hand.hand();
```

Comment:
This is quite unexpected. I don't know yet what the Hand is supposed to do, but if I'm looking this in this project & file context, `Hand.hand()` does not make any sense..